tests/Config/RedisConnectionConfigTest.php

```
@@ -125,28 +124,43 @@ public function testConstructorThrowsExceptionWhenRetryIntervalIsNegative(): voi
+        self::assertSame($expected, $actual);
+    }
+
+    /**
+     * @return array<string, array{RedisConnectionConfig, callable, mixed}>
+     */
+    public static function zeroValueProvider(): array
+    {
+        return [
+            'timeout' => [
+                new RedisConnectionConfig('localhost', 6379, 0.0),
+                static fn (RedisConnectionConfig $c): float => $c->getTimeout(),
```

PHPMD (2.15.0)

Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)